hack/make-rules/test-cmd.sh

```diff
@@ -39,19 +39,24 @@ function run_kube_apiserver() {
   # Include RBAC (to exercise bootstrapping), and AlwaysAllow to allow all actions
   AUTHORIZATION_MODE="RBAC,AlwaysAllow"
 
+  # secure port will be used to test authz/authn/impersonation features
+  TOKEN_AUTH_FILE=${KUBE_ROOT}/hack/make-rules/test-cmd-token-auth-file.csv
+
   "${KUBE_OUTPUT_HOSTBIN}/kube-apiserver" \
     --address="127.0.0.1" \
     --public-address-override="127.0.0.1" \
     --port="${API_PORT}" \
     --authorization-mode="${AUTHORIZATION_MODE}" \
+    --secure-port="${SECURE_API_PORT}" \
     --admission-control="${ADMISSION_CONTROL}" \
     --etcd-servers="http://${ETCD_HOST}:${ETCD_PORT}" \
     --public-address-override="127.0.0.1" \
     --kubelet-port=${KUBELET_PORT} \
     --runtime-config=api/v1 \
     --storage-media-type="${KUBE_TEST_API_STORAGE_TYPE-}" \
     --cert-dir="${TMPDIR:-/tmp/}" \
-    --service-cluster-ip-range="10.0.0.0/24" 1>&2 &
+    --service-cluster-ip-range="10.0.0.0/24" \
+    --token-auth-file=${TOKEN_AUTH_FILE} 1>&2 &
```

Comment:
`--insecure-allow-any-token` flag instead of a token file accepts a token in the form "user/group1,group2", and is useful for testing lots of different users without needing a token file